components/script/dom/htmlelement.rs

```diff
@@ -272,9 +272,13 @@ impl HTMLElementMethods for HTMLElement {
 
     // https://drafts.csswg.org/cssom-view/#dom-htmlelement-offsetparent
     fn GetOffsetParent(&self) -> Option<Root<Element>> {
-        if self.is::<HTMLBodyElement>() || self.is::<HTMLHtmlElement>() {
+        if self.is::<HTMLHtmlElement>() {
```

Comment:
This incorrect, an `html` element can be the child of whatever. The test really needs to be about the root element.